README.md

```diff
@@ -230,6 +230,10 @@ A value assigned to the field.  Optional.
 
 Textual annotation of a field.  Clients may use this as a label.  Optional.
 
+####cardinality
```

Comment:
Initial cap? Do you mean that it should be `####Cardinality` instead of `####cardinality`? If so, wouldn't that deviate from that all lowercase pattern being used by the other attributes?